service/controller/v11/adapter/auto_scaling_group.go

```diff
@@ -41,7 +41,7 @@ func workerCountRatio(workers int, ratio float32) string {
 	value := float32(workers) * ratio
 	rounded := int(value + 0.5)
 
-	if workers > 0 && rounded == 0 {
+	if rounded == 0 {
```

Comment:
This is to fix updates when there is only one worker. The number computed here was formerly `0` in certain cases which renders the CF templates invalid. Let me know when there was a deeper meaning or purpose behind this.